extensions/wikia/WikiaMobile/WikiaMobileService.class.php

```diff
@@ -216,17 +216,21 @@ private function handleContent($content = ''){
 		wfProfileOut( __METHOD__ );
 	}
 
+	private function isArticleView() {
+		$action = $this->wg->Request->getVal( 'action', 'view' );
+		$nameSpace = $this->wg->Title->getNamespace();;
+
+		return ( ( $action == 'view' || $action == 'ajax' ) &&
+			$this->wg->Title->getArticleId() != 0 &&
```

Comment:
consider using global $wgTitle - you are accessing it twice, it might be a little bit more readable